agentPresence/agentPresenceController.js

```diff
@@ -0,0 +1,111 @@
+var micc = new Micc('localhost');
+
+function login() {
+    micc = new Micc($('#miccServer').val());
+    micc.login(
+        $('#username').val(),
+        $('#password').val(),
+        connectToEmployeeHub
+    );
+}
+
+function connectToEmployeeHub(data) {
+    populateEmployeeInfo();   
+    micc.getEmployeeBusyReasonCodes('me', populateBusyReasonCodes);
+
+    var singlarConnection = $.hubConnection(`${data.miccServer}/miccsdk/`, {
```

Comment:
This is now the second signalR example and is directly copied from the first.  The SignalR connection piece should be refactored and pulled into a common file much like has been done for micc.js.  I suggest MiccSignalR as a name.  It would hold the connection, create hubs, and wrap hub functions.